browser-test/src/applicant/northstar_applicant_question_translation.test.ts

```diff
@@ -0,0 +1,204 @@
+import {test, expect} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  logout,
+  selectApplicantLanguage,
+  validateScreenshot,
+} from '../support'
+
+test.describe('Admin can manage translations', {tag: ['@northstar']}, () => {
+  test.beforeEach(async ({page}) => {
+    await enableFeatureFlag(page, 'north_star_applicant_ui')
+  })
+
+  test('Expect single-answer question is translated for applicant', async ({
+    page,
+    adminPrograms,
+    adminQuestions,
+    adminTranslations,
+    applicantQuestions,
+  }) => {
+    await loginAsAdmin(page)
+
+    // Add a new question to be translated
+    const questionName = 'name-translated'
+    await adminQuestions.addNameQuestion({questionName})
+
+    // Go to the question translation page and add a translation for Spanish
+    await adminQuestions.goToQuestionTranslationPage(questionName)
+    await adminTranslations.selectLanguage('Spanish')
+    await adminTranslations.editQuestionTranslations(
+      'ingrese\n1.nombre\n2.segundo nombre\n3.apellido',
```

Comment:
exist

Reply:
Yeah, program cards in North Star do not have a "Program Details"/"Detalles del programa" button.